### ISSUE TITLE
Chart Lava Short Code - CurvedLines:'false' does not function

### ISSUE THREAD

**Issue by randyjreed** [SparkDevNetwork/Rock#3177]
### Prerequisites
* [X] Put an X between the brackets on this line if you have done all of the following:
  * Can you reproduce the problem on a fresh install or the [demo site ](http://rock.rocksolidchurchdemo.com/)?
  * Did you include your Rock version number and [client culture ](https://github.com/SparkDevNetwork/Rock/wiki/Environment-and-Diagnostics-Information) setting?
  * Did you [?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ASparkDevNetwork%2FSlack perform a cursory search ](https://github.com/issues) to see if your bug or enhancement is already reported?

### Description
When using the lava short code for Chart and using the curvedlines:'false' the lines are still curved. I have tested this on the demo site on 7.4 also.

### Steps to Reproduce
1. Add HTML block to a page
1. Add the following code to the block
`{[ chart type:'line' labels:'1/1/2017,2/1/2017,6/1/2017, 7/1/2017, 8/1/2017, 9/1/2017' xaxistype:'time' curvedlines:'false' ]}
 [[ dataset label:'Small Groups' data:'12, 15, 34, 34, 15, 5' fillcolor:'#059BFF' curvedlines:'false' ]] [[ enddataset ]]
 [[ dataset label:'Serving Teams' data:'10, 22, 41, 41, 22, 5' fillcolor:'#FF3D67' curvedlines:'false' ]] [[ enddataset ]]
 [[ dataset label:'General Groups' data:'5, 12, 21, 21, 12, 5' fillcolor:'#4BC0C0' curvedlines:'false' ]] [[ enddataset ]]
 [[ dataset label:'Fundraising Groups' data:'3, 17, 32, 32, 17, 5' fillcolor:'#FFCD56' curvedlines:'false' ]] [[ enddataset ]]
{[ endchart ]}`
1. Reload the page and lines will be curved.

**Expected behavior:**

I expected by using the curvedlines:'false' option the lines would be straight on the line graph.

**Actual behavior:**

Lines are curved as if i did not set the option to false.

### Versions
* **Rock Version:** [7.4]
* **Client Culture Setting:** [en-US]

**Comment by JimMichael** [comment]
I would add that it's not as simple as `curvedlines:'false'` not working _at all). The demo data
```
{[ chart type:'line' curvedlines:'false' ]}
    [[ dataitem label:'Small Groups' value:'45' ]] [[ enddataitem ]]
    [[ dataitem label:'Serving Groups' value:'38' ]] [[ enddataitem ]]
    [[ dataitem label:'General Groups' value:'34' ]] [[ enddataitem ]]
    [[ dataitem label:'Fundraising Groups' value:'12' ]] [[ enddataitem ]]
{[ endchart ]}
```
shows straight lines just fine... so it must be something buggy with the number of parameters, or their order, etc.

**Comment by randyjreed** [comment]
Yes, i see you added this to the demo site. It appears the difference is that the `xaxistype:'time'`.

**Comment by randyjreed** [comment]
I removed the xaxistype:'time' and it still does not function. The only other difference is the use of `dataset` to allow multiple data points as apposed to the `dataitem`

**Comment by garrettjohnson** [comment]
Fixed in 8.6 with https://github.com/SparkDevNetwork/Rock/commit/ade09a092401a11852877943b19bd2a5d9fba7e5
